api/routes/apds/post.test.js

```diff
@@ -64,6 +64,8 @@ tap.test('apds POST endpoint', async endpointTest => {
       test.ok(res.end.calledOnce, 'response is terminated');
     });
 
+    // Why is a 500 status being returned when the frontend sends malformed data?
+    // Should this not be a 4xx status? What the heck?
     tests.test(
       'sends a 500 if the newly-generated APD fails schema validation',
```

Comment:
If the front-end sends bad data, I would expect the api to respond with a 4xx HTTP status code (client error), and not a 5xx HTTP status code (server error).

Looking at the rest of the API implementation, it seems like this was an error in judgement that was made early on, and we ran with it. I'm going to file a tech debt ticket for this, but likely we will not be able to address this until after ATO. Any thoughts on this, @thetif?